tmt/steps/__init__.py

```diff
@@ -734,6 +734,35 @@ def command(cls) -> click.Command:
                 command = option(command)
             commands[method.name] = command
 
+            for param in command.params:
+                if param.name == 'how':
+                    continue
+
+                if not isinstance(param, click.Option):
+                    continue
+
+                assert command.name is not None  # narrow type
+                assert param.name is not None  # narrow type
+
+                command_name, method_name, param_name = \
+                    command.name.upper(), method.name.upper(), param.name.upper()
+
+                envvar = f'TMT_PLUGIN_{command_name}_{method_name}_{param_name}'
+
+                if param.envvar is None:
+                    param.envvar = [envvar]
+
+                elif isinstance(param.envvar, str):
+                    param.envvar = [param.envvar, envvar]
+
+                elif isinstance(param.envvar, list):
+                    param.envvar.append(envvar)
+
+                else:
+                    assert False, \
```

Comment:
Even if this feels like a programming error, I would rather raise an error, and not use assert for error handling.

Reply:
Absolutely, this is a development left-over. An exception is needed here.

Reply:
Fixed in e88cadc2